packages/realm-app-importer/package.json

```diff
@@ -0,0 +1,49 @@
+{
+    "name": "realm-app-importer",
+    "version": "0.1.0",
+    "description": "Implements cross-platform fetching used by Realm JS",
+    "main": "dist/index.js",
+    "types": "dist/index.d.ts",
+    "bin": "realm-app-importer",
+    "scripts": {
+      "build": "../../node_modules/.bin/tsc -p tsconfig.build.json",
+      "tslint": "../../node_modules/.bin/tslint -p .",
+      "prepack": "rm -rf dist && npm run build",
+      "start": "ts-node src/cli.ts"
+    },
+    "files": [
+      "dist",
+      "templates",
+      "realm-app-importer"
+    ],
+    "author": {
```

Comment:
Please add `contributors` with your name in the array :smile:

Reply:
👍